R/install-dev.sh

```diff
@@ -38,7 +38,12 @@ pushd $FWDIR > /dev/null
 if [ ! -z "$R_HOME" ]
   then
     R_SCRIPT_PATH="$R_HOME/bin"
-   else
+  else
+    # if system wide R_HOME is not found, then exit
+    if ! [ `command -v R` ]; then
```

Comment:
Yeah, we typically put the `!` after the test:
* https://github.com/apache/spark/blob/6d506c9ae9a2519d1a93e788ae5316d4f942d35d/dev/lint-python#L44
* https://github.com/apache/spark/blob/6d506c9ae9a2519d1a93e788ae5316d4f942d35d/dev/lint-java#L25

(In Bash, `[ ... ]` and `test` are synonyms.)